### PR TITLE
TEST/JENKINS: Use correct options for ucx_perftest

### PR DESCRIPTION
**Pull request by brminich**
## What
Use UCX env variables instead of -d and -x options for UCP perftest.

## Why ?
- To avoid flooding of warnings like: 
`UCX  WARN  -d 'posix' ignored for UCP test; see NOTES section in help message`
- To run UCP tests with different devices rather than running with the same set of devices on each cycle iteration

### CODE REVIEW

File: contrib/test_jenkins.sh [hunk 1]
```diff
@@ -574,10 +574,15 @@ run_ucx_perftest_mpi() {
 	# hack for perftest, no way to override params used in batch
 	# todo: fix in perftest
 	sed -s 's,-n [0-9]*,-n 1000,g' $ucx_inst_ptest/msg_pow2 | sort -R > $ucx_inst_ptest/msg_pow2_short
-	cat $ucx_inst_ptest/test_types | grep -v cuda | sort -R > $ucx_inst_ptest/test_types_short
+	cat $ucx_inst_ptest/test_types | grep -v cuda | grep -v ucp | sort -R > $ucx_inst_ptest/test_types_short_uct
```
Comment (yosefe) [inline contrib/test_jenkins.sh]: maybe split '$ucx_inst_ptest/test_types' to $ucx_inst_ptest/test_types_uct and $ucx_inst_ptest/test_types_ucp?
to not rely on test name to decide if it's ucp/uct test

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8390/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8392/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8397/ for details (Mellanox internal link).

**Comment by brminich** [comment]
@yosefe, plz check

File: Makefile.am [hunk 2]
```diff
@@ -20,7 +20,7 @@ doc_dir = $(pkgdatadir)/doc
 
 if !DOCS_ONLY
 perftest_dir = $(pkgdatadir)/perftest
-dist_perftest__DATA = contrib/ucx_perftest_config/msg_pow2 contrib/ucx_perftest_config/README contrib/ucx_perftest_config/test_types contrib/ucx_perftest_config/transports
+dist_perftest__DATA = contrib/ucx_perftest_config/msg_pow2 contrib/ucx_perftest_config/README contrib/ucx_perftest_config/test_types_uct contrib/ucx_perftest_config/test_types_ucp  contrib/ucx_perftest_config/transports
```
Comment (yosefe) [inline Makefile.am]: let's split to multiple lines with \ ?

File: contrib/test_jenkins.sh [hunk 3]
```diff
@@ -589,27 +594,32 @@ run_ucx_perftest_mpi() {
 	do
 		if [[ $ucx_dev =~ .*mlx5.* ]]; then
 			opt_transports="-b $ucx_inst_ptest/transports"
+			tls=`awk '{print $3 }' $ucx_inst_ptest/transports | tr '\n' ',' | sed -r 's/,$//; s/mlx5/x/g'`
```
Comment (yosefe) [inline contrib/test_jenkins.sh]: does this convert transports batch file to a list?

Comment (brminich) [reply contrib/test_jenkins.sh]: yes

File: Makefile.am (hunk 2 above)
Comment (brminich) [reply Makefile.am]: ok

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8399/ for details (Mellanox internal link).

**Pull request merged**